OpenRA.Mods.Common/Traits/World/EditorActorLayer.cs

```diff
@@ -106,7 +106,8 @@ EditorActorPreview Add(string id, ActorReference reference, bool initialSetup =
 
 			var preview = new EditorActorPreview(worldRenderer, id, reference, owner);
 			previews.Add(preview);
-			screenMap.Add(preview, preview.Bounds);
+			if (!preview.Bounds.IsEmpty)
```

Comment:
We silently skip in the actor selector because we don't have a declarative way to say which actors should be there and which shouldn't.  We do have a declarative way of saying which actors should be in the map, and shouldn't tolerate cases where invalid actors are defined there.